docs/turn-howto.md

```diff
@@ -42,65 +38,112 @@ This will install and start a systemd service called `coturn`.
+See Docker documentation for details.
+
+For Docker all coturn configuration options listed below can also be specified
+on the docker startup script, like this:
+
+        [... same as above to start ...]
+        --restart=unless-stopped instrumentisto/coturn \
+        --total-quota=1200 --realm=<your turn server domain, ie turn.domain.tld> \
+        --external-ip='$(detect-external-ip)'
+
+So any configuration option described below you can add after the docker image
+name, preceeded by --. You can also mount a volume with -v and place a
+configuration file there, for example turnserver.conf.
+
+
 ### Configuration
 
 1.  Create or edit the config file in `/etc/turnserver.conf`. The relevant
     lines, with example values, are:
 
         use-auth-secret
-        static-auth-secret=[your secret key here]
+        static-auth-secret=<your secret key here>
         realm=turn.myserver.org
+        external-ip='$(detect-external-ip)'
+        no-cli
+
+    The 'external-ip' can either be the included detection script like this example
+    or the real external IP. That IP can be different from the local / relay IP, which
+    is auto detected and not specified.
 
     See `turnserver.conf` for explanations of the options. One way to generate
     the `static-auth-secret` is with `pwgen`:
 
         pwgen -s 64 1
 
-    A `realm` must be specified, but its value is somewhat arbitrary. (It is
-    sent to clients as part of the authentication flow.) It is conventional to
-    set it to be your server name.
-
-1.  You will most likely want to configure coturn to write logs somewhere. The
-    easiest way is normally to send them to the syslog:
+2.  You will most likely want to configure coturn to write logs somewhere. The easiest way is normally to send them to the syslog:
 
         syslog
 
-    (in which case, the logs will be available via `journalctl -u coturn` on a
-    systemd system). Alternatively, coturn can be configured to write to a
-    logfile - check the example config file supplied with coturn.
+    (in which case, the logs will be available via journalctl -u coturn on a systemd system). Alternatively, coturn can be configured to write to a logfile - check the example config file supplied with coturn.
 
-1.  Consider your security settings. TURN lets users request a relay which will
+3.  Consider your security settings. TURN lets users request a relay which will
     connect to arbitrary IP addresses and ports. The following configuration is
     suggested as a minimum starting point:
-
-        # VoIP traffic is all UDP. There is no reason to let users connect to arbitrary TCP endpoints via the relay.
+    
+        # VoIP traffic is all UDP. There is no reason to let users connect to arbitrary TCP endpoints
+        # via the relay.
         no-tcp-relay
+        # Require authentication (same as for TURN) for STUN requests to preven refrection attacks
+        # via the STUN mechanism.
```

Comment:
```suggestion
        # Require authentication (same as for TURN) for STUN requests to prevent reflection attacks
        # via the STUN mechanism.
```